modules/EasyAbp.EShop.Stores/src/EasyAbp.EShop.Stores.Domain/EasyAbp/EShop/Stores/Stores/StoreManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using EasyAbp.EShop.Stores.StoreOwners;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Services;
+
+namespace EasyAbp.EShop.Stores.Stores
+{
+    public class StoreManager : DomainService, IStoreManager
+    {
+        private readonly IStoreRepository _storeRepository;
+        private readonly IStoreOwnerRepository _storeOwnerRepository;
+
+        public StoreManager(IStoreRepository storeRepository,
+            IStoreOwnerRepository storeOwnerRepository)
+        {
+            _storeRepository = storeRepository;
+            _storeOwnerRepository = storeOwnerRepository;
+        }
+
+        public async Task<Store> CreateAsync(Store store, IEnumerable<Guid> ownerIds = null)
+        {
+            store = await _storeRepository.InsertAsync(store);
+
+            await UpdateStoreOwnersAsync(store.Id, ownerIds);
+
+            return store;
+        }
+
+        public async Task<Store> UpdateAsync(Store store, IEnumerable<Guid> ownerIds = null)
+        {
+            store = await _storeRepository.UpdateAsync(store);
+
+            await UpdateStoreOwnersAsync(store.Id, ownerIds);
+
+            return store;
+        }
+
+        protected virtual async Task UpdateStoreOwnersAsync(Guid storeId, IEnumerable<Guid> ownerIds)
```

Comment:
It will be easier to manage if the StoreOwner has its own application service and management pages (remember it is an aggregate root). Then the `StoreManager` can be removed.